docs/source/executing-operations/demand-control.mdx

```diff
@@ -0,0 +1,554 @@
+
+|           | Mutation | Query | Subscription |
+| --------- | -------- | ----- | ------------ |
+| Object    | 1        | 1     | 1            |
+| Interface | 1        | 1     | 1            |
+| Union     | 1        | 1     | 1            |
+| Scalar    | 0        | 0     | 0            |
+| Enum      | 0        | 0     | 0            |
+
+For example, assume the following query gets a response with six products and ten reviews:
+
+```graphql
+query ExampleQuery {
+  topProducts {
+    name
+    reviews {
+      author {
+        name
+      }
+    }
+  }
+}
+```
+
+Assuming each review having exactly one author, the total cost of the query is 26.
+
+<ExpansionPanel title="Example query's cost calculation">
+
+```text disableCopy=true showLineNumbers=false
+1 Query (0 cost) + 6 product objects (6) + 6 name scalars (0) + 10 review objects (10) + 10 author objects (10) + 10 name scalars (0) = 26 total cost
+```
+
+</ExpansionPanel>
+
+### Estimated and actual costs
+
+For an operation with list fields, the router must run the operation to get the actual number of items in its lists. Without actual list sizes, the cost of an operation can only be estimated before it's executed, where you assume the size of lists. 
+
+After an operation is executed, the actual cost per operation can be calculated with the actual list sizes.
+
+<Note>
+
+The difference between estimated and actual operation cost calculations is due only to the difference between assumed and actual sizes of list fields.
+
+</Note>
+
+### Measurement and enforcement modes
+
+When rolling out demand control, you first need to gather information about the queries that are already being executed against your graph so you can decide when to reject requests.
+
+The router's demand control supports a measurement mode that enables you to gather this information without impacting your running services. You can define telemetry instruments to monitor your operations and decide on their maximum cost threshold.  
+
+After gathering enough data, you can then configure your router with maximum cost and list size limits and set demand control to enforcement mode, where it rejects operations with costs exceeding the limit.
+
+## Configuring demand control
+
+To enable demand control in the router, configure the `preview_demand_control` option in `router.yaml`:
+
+```yaml title="Example demand control configuration"
+preview_demand_control:
+  enabled: true
+  mode: measure
+  strategy:
+    static_estimated:
+      list_size: 10 
+      max: 1000 
+```
+
+When `preview_demand_control` is enabled, the router measures the cost of each operation and can enforce operation cost limits, based on additional configuration.  
+
+Customize `preview_demand_control` with the following settings:
+
+| Option              | Valid values            | Default value | Description                                                                                          |
```

Comment:
- The options seem like they should be in code font. (enabled, mode, strategy, static_estimated.list_size, etc.)

- For options without default values, it would be clearer to use `none` or `--` to denote there isn't any.

- Some descriptions have periods, others do not.